vitrina/classifiers/models.py

```diff
@@ -1,27 +1,54 @@
 from django.db import models
-from treebeard.al_tree import AL_Node
+from treebeard.mp_tree import MP_Node, MP_NodeManager
+
+
+class Category(MP_Node):
+    BOOKS = 'books'
+    ANALYSIS = 'analysis'
+    ELECTRICITY = 'electricity'
+    CITYSCAPE = 'cityscape'
+    ACCOUNTING = 'accounting'
+    AIRPLANE = 'airplane'
+    BALANCE = 'balance'
+    MARKETING = 'marketing'
+    NATURE = 'nature'
+    OIL = 'oil'
+    PANTHEON = 'pantheon'
+    RETIREMENT = 'retirement'
+    TRUMPET = 'trumpet'
+    VACCINE = 'vaccine'
+    ICONS = [
+        (BOOKS, 'books'),
+        (ANALYSIS, 'analysis'),
+        (ELECTRICITY, 'electricity'),
+        (CITYSCAPE, 'cityscape'),
+        (ACCOUNTING, 'accounting'),
+        (AIRPLANE, 'airplane'),
+        (BALANCE, 'balance'),
+        (MARKETING, 'marketing'),
+        (NATURE, 'nature'),
+        (OIL, 'oil'),
+        (PANTHEON, 'pantheon'),
+        (RETIREMENT, 'retirement'),
+        (TRUMPET, 'trumpet'),
+        (VACCINE, 'vaccine'),
+    ]
```

Comment:
Do not hardcode few possible icons, instead provide a link to a full list of icon names.

Reply:
Currently the icons are all in the icons.css file, I listed them all out in the model, how would I go about automating the process of providing icons from a .css file to the choices field ?

Reply:
Maybe we can use icons as documented in Bulma and provide link to [Font Awesome search page](https://fontawesome.com/search)? This way, when adding a new category, we could pick form this library, by specifying a string like `home`, which would translate into `fa fa home` css class.

Reply:
The icons that are used are not part of font awesome, they are custom as I understand. There are 14 of them in total.